CHANGELOG.md

```diff
@@ -1,4 +1,15 @@
-## 1.9.1 (Unreleased)
+## 1.10.0 (Unreleased)
+
+IMPROVEMENTS:
+feat: add ability to import `newrelic_synthetics_monitor` ([#267](https://github.com/terraform-providers/terraform-provider-newrelic/pull/267))
```

Comment:
Missing the `*` at the beginning so these are formatted as a list 😉

Reply:
👍